### PR TITLE
doc improvement: General doc fixes

### PR DESCRIPTION
**Pull request by umapraseeda**
NCSDK-16811
General doc fixes

Signed-off-by: UMA PRASEEDA <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/libraries/networking/nrf_cloud_cell_pos.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ This library is an enhancement to the :ref:`lib_nrf_cloud` library.
 Configuration
 *************
 
-Configure the following options to enable or disable the use of this library:
+Configure the following option to enable or disable the use of this library:
```
Comment (peknis) [inline doc/nrf/libraries/networking/nrf_cloud_cell_pos.rst:19]:
```suggestion
Configure the following Kconfig option to enable or disable the use of this library:
```

File: doc/nrf/libraries/others/app_event_manager.rst [hunk 2]
```diff
@@ -329,7 +329,7 @@ Tracing hooks
 .. em_tracing_hooks_start
 
 The Application Event Manager uses flexible mechanism to implement hooks when an event is submitted, before it is processed, and after its processing.
-Oryginally designed to implement event tracing, the tracing hooks can be used for other purposes as well.
+Originally designed to implement event tracing, the tracing hooks can be used for other purposes as well.
```
Comment (peknis) [inline doc/nrf/libraries/others/app_event_manager.rst:332]:
```suggestion
The tracing hooks are originally designed to implement event tracing, but you can use them for other purposes as well.
```

File: samples/bluetooth/central_bas/README.rst [hunk 3]
```diff
@@ -28,6 +21,13 @@ The sample supports the following development kits:
 
 The sample also requires a device running a BAS Server to connect with (for example, another development kit running the :ref:`peripheral_hids_mouse` or :ref:`peripheral_hids_keyboard` sample, or a Bluetooth® Low Energy dongle and nRF Connect for Desktop).
 
+Overview
+********
+
+When connected, the sample subscribes to battery level notifications.
+Every notification that is received is printed to the terminal.
+If the device does not support notifications for the Battery Level Characteristic, you can request to read the battery level by pressing Button 1.
```
Comment (peknis) [inline samples/bluetooth/central_bas/README.rst:29]:
```suggestion
If the device does not support notifications for the Battery Level Characteristic, press **Button 1** to request for reading the battery level.
```

File: samples/bluetooth/central_hr_coded/README.rst [hunk 4]
```diff
@@ -29,6 +22,13 @@ The sample supports the following development kits:
 The sample also requires a device running a Heart Rate Server with LE Coded PHY support to connect to.
 For example, another development kit running the :ref:`peripheral_hr_coded` sample.
 
+Overview
+********
+
+The sample demonstrates a Bluetooth® LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support, which is not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
+It then establishes a connection to the first Peripheral device in range.
+It can be used together with the :ref:`peripheral_hr_coded` sample.
```
Comment (peknis) [inline samples/bluetooth/central_hr_coded/README.rst:30]:
```suggestion
The sample demonstrates a Bluetooth® LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support, which is not available in Zephyr Bluetooth LE Controller.
See :ref:`ug_ble_controller` for more information.
It then establishes a connection to the first peripheral device in range.
You can use it together with the :ref:`peripheral_hr_coded` sample.
```

File: samples/bluetooth/central_smp_client/README.rst [hunk 5]
```diff
@@ -39,6 +28,17 @@ The sample also requires a device running `mcumgr`_ with transport protocol over
 .. note::
    This sample does not program the device using DFU.
 
+Overview
+********
+
+After connecting, the sample starts MTU size negotiation, discovers the GATT database of the server, and configures the DFU SMP Client.
+When configuration is complete, the sample is ready to send SMP commands.
+
+To send an echo command, press Button 1 on the development kit.
```
Comment (peknis) [inline samples/bluetooth/central_smp_client/README.rst:37]:
```suggestion
To send an echo command, press **Button 1** on the development kit.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/bluetooth/enocean/README.rst [hunk 6]
```diff
@@ -30,6 +21,15 @@ The sample also requires at least one :ref:`supported EnOcean device <bt_enocean
 .. note::
     The sample supports up to four devices at a time that work with one kit.
 
+Overview
+********
+
+The EnOcean sample sets up a basic Bluetooth observer for both EnOcean switches and sensors.
+
+The observer device forwards incoming advertisements to the EnOcean library for processing.
+The application receives events from the EnOcean library through callbacks, and prints the outcome to console.
```
Comment (peknis) [inline samples/bluetooth/enocean/README.rst:30]:
```suggestion
The application receives events from the EnOcean library through callbacks, and prints the outcome to the console.
```

File: samples/bluetooth/llpm/README.rst [hunk 7]
```diff
@@ -10,6 +10,18 @@ Bluetooth: LLPM
...
 You can use it to determine the transmission latency of LLPM-enabled connections, or to compare with different connection parameters and check their influence on the results.
 
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also supports other development kits running SoftDevice Controller variants that support LLPM (see :ref:`nrfxlib:softdevice_controller` Proprietary feature support).
```
Comment (peknis) [inline samples/bluetooth/llpm/README.rst:20]:
```suggestion
The sample also supports other development kits running SoftDevice Controller variants that support LLPM (see :ref:`nrfxlib:softdevice_controller` for the proprietary feature support).
```
Not 100% about this, but just "Proprietary feature support" felt a bit odd.

File: scripts/hid_configurator/README.rst [hunk 8]
```diff
@@ -82,7 +82,7 @@ Complete the following steps:
        In versions earlier than 5.44, the HID device attached by BlueZ could obtain wrong VID and PID values (ignoring values in Device Information Service), which would stop HIDAPI from opening the device.
        In versions earlier than 5.56, the HID device attached by BlueZ might provide incomplete HID feature report on get operation.
 
-#. If you don't want to use the root access to run the python script, copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d` and reconnect the device.
+#. If you do not want to use the root access to run the python script, copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d` and reconnect the device.
```
Comment (peknis) [inline scripts/hid_configurator/README.rst:85]:
```suggestion
#. If you do not want to use the root access to run the Python script, copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d` and reconnect the device.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: applications/nrf_desktop/README.rst [hunk 9]
```diff
@@ -1493,7 +1503,7 @@ The central must also control the Bluetooth connection parameters.
 These features are implemented by the following application modules:
 
 * :ref:`nrf_desktop_ble_scan` - Controls the Bluetooth scanning.
-* :ref:`nrf_desktop_ble_conn_params` - Control the Bluetooth connection parameters and react on connection slave latency update requests received from the connected peripherals.
+* :ref:`nrf_desktop_ble_conn_params` - Controls the Bluetooth connection parameters and reacts on connection slave latency update requests received from the connected peripherals.
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply applications/nrf_desktop/README.rst:1506]: @divipillai, could you check with the SME?

[the remark this answers is not in the document]
Comment (divipillai) [reply applications/nrf_desktop/README.rst:1506]: Checked with @MarekPieta and he confirmed that "latency update requests" should be fine.

File: doc/nrf/libraries/bluetooth_services/mesh/light_xyl.rst [hunk 10]
```diff
@@ -7,6 +7,8 @@ Light xyL models
    :local:
    :depth: 2
 
+:c:func:`bt_gatt_exchange_mtu`
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply doc/nrf/libraries/bluetooth_services/mesh/light_xyl.rst:10]: Stale entry, removing.

File: samples/bluetooth/llpm/README.rst (hunk 7 above)
Comment (umapraseeda) [reply samples/bluetooth/llpm/README.rst:20]: I think it was meant to link to the Proprietary feature support table here:  http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8637/nrfxlib/softdevice_controller/README.html#softdevice-controller. I shall try to add a link.

Comment (peknis) [reply samples/bluetooth/llpm/README.rst:20]: Yep, I agree it points to that table. I guess it could work with either the original or my change.

Comment (umapraseeda) [reply samples/bluetooth/llpm/README.rst:20]: Adding a link here: https://github.com/nrfconnect/sdk-nrfxlib/pull/825/files

**Review by peknis — APPROVED** [review]

**Comment by umapraseeda** [comment]
Thanks for the review, @peknis !

**Review by trantanen — APPROVED** [review]
Modem shell is fine

**Review by edmont — APPROVED** [review]
Fine with openthread changes.

**Review by alexsven — APPROVED** [review]
Looks good for nrf5340_audio

File: include/net/nrf_cloud_agps.h [hunk 11]
```diff
@@ -58,7 +58,7 @@ int nrf_cloud_agps_process(const char *buf, size_t buf_len);
 /**@brief Query which A-GPS elements were actually received
  *
  * @param received_elements return copy of requested elements received
- * since agps request made
+ * since AGPS request was made
```
Comment (jayteemo) [inline include/net/nrf_cloud_agps.h:61]:
```suggestion
 * since A-GPS request was made
```

**Review by jayteemo — APPROVED** [review]

**Review by plskeggs — APPROVED** [review]

File: west.yml [hunk 12]
```diff
@@ -112,7 +112,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v2.1.0-rc1
+      revision: pull/825/head
```
Comment (rlubos) [inline west.yml:115]: Please update SHA: `ee0d1c66944f15a47d08883c4826b3b4b4cb317e`

**Review by b-gent — APPROVED** [review]

File: include/net/nrf_cloud_agps.h (hunk 11 above)
Comment (umapraseeda) [reply include/net/nrf_cloud_agps.h]: Fixed

File: west.yml (hunk 12 above)
Comment (umapraseeda) [reply west.yml]: updated

**Review by rlubos — APPROVED** [review]

**Comment by rlubos** [comment]
Confirmed that TFM/Zigbee failures are not related, merging.

**Pull request merged**